### PR TITLE
add reset password

### PR DESCRIPTION
**Pull request by ValeriyaTarasenko**

### CODE REVIEW

File: TennisBusiness/View controllers/LoginViewController.swift [hunk 1]
```diff
@@ -65,10 +85,24 @@ class LoginViewController: ScreenViewController, UITextFieldDelegate {
...
+        
+        keyboardFrame = self.view.convert(keyboardFrame, from: nil)
+        var contentInset = self.scrollView.contentInset
+        contentInset.bottom = keyboardFrame.size.height + Constants.scrollInset + TextField.Constants.bottomLineInset
+        scrollView.contentInset = contentInset
+    }
+    
+    @objc private func keyboardWillBeHidden(notification: NSNotification) {
+        let contentInset: UIEdgeInsets = .zero
+        scrollView.contentInset = contentInset
```
Comment (NikolayMikhailishin) [inline TennisBusiness/View controllers/LoginViewController.swift]: It should be work also as `scrollView.contentInset = .zero`

File: TennisBusiness/View controllers/ResetPasswordViewController.swift [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+       
+       keyboardFrame = self.view.convert(keyboardFrame, from: nil)
+       var contentInset = self.scrollView.contentInset
+       contentInset.bottom = keyboardFrame.size.height + Constants.scrollInset + TextField.Constants.bottomLineInset
+       scrollView.contentInset = contentInset
+    }
+
+    @objc private func keyboardWillBeHidden(notification: NSNotification) {
+       let contentInset: UIEdgeInsets = .zero
+       scrollView.contentInset = contentInset
```
Comment (NikolayMikhailishin) [inline TennisBusiness/View controllers/ResetPasswordViewController.swift]: Same here

File: TennisBusiness/View controllers/ResetPasswordViewController.swift [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+    }
+
```
Comment (NikolayMikhailishin) [inline TennisBusiness/View controllers/ResetPasswordViewController.swift]: Please add an empty line here

File: TennisBusiness/View controllers/ResetPasswordViewController.swift [hunk 4]
```diff
@@ -0,0 +1,115 @@
...
+           showErrorMessageKey("KEY_ERROR_WRONG_EMAIL")
+           return
+        }
+        authDataManager.resetPassword(with: mail)
+        .then { [weak self] () -> Void in
+            self?.dismiss(animated: true, completion: nil)
+        }
+        .catch { [weak self] error in
+            self?.showErrorMessage(error.localizedDescription)
+        }
```
Comment (NikolayMikhailishin) [inline TennisBusiness/View controllers/ResetPasswordViewController.swift]: Please align vertically this block.

File: TennisBusiness/View controllers/ResetPasswordViewController.swift [hunk 5]
```diff
@@ -0,0 +1,115 @@
...
+    }
+    
+    @IBAction private func closeButtonTouchUpInside(_ sender: Any) {
+        dismiss(animated: true, completion: nil)
+    }
+
```
Comment (NikolayMikhailishin) [inline TennisBusiness/View controllers/ResetPasswordViewController.swift]: please remove this line

**Review by MykhailishynMykola — APPROVED** [review]

**Pull request merged**